app/models/organization.rb

```diff
@@ -75,7 +75,7 @@ class Organization < ActiveRecord::Base
   after_commit :update_associations
 
   geocoded_by :search_location
-  after_validation :geocode
+  after_validation :geocode, if: ->(org) { org.search_location.present? }
```

Comment:
We still need them geocoded for the map (`/where`)

Reply:
Maybe not, because that's locations? But I still think we want it.

Reply:
The `if` here is meant to skip the geocoding API call if `search_location` isn't set, since we won't be able to geolocate without it: https://github.com/alexreisner/geocoder#avoiding-unnecessary-api-requests

I added some documentation and a check for the component location fields having been changed in 35588ce3 -- let me know if I'm missing something though.